### PR TITLE
Some work on #4

### PR DESCRIPTION
**Pull request by stisa**
Hi all, I had some free time ( read: useful procrastination ) so I tried my hand at issue #4 : 
- **sponsors page** uses the csv files in _data to generate the lists. I had to remove spaces from the headers of the columns to make it work, we may need to fix the script that generates them;
- **community** basically a copy paste from the current one, could use some more content;
- **documentation** same as above
- **download**:
  -  `windows`,`linux-bsd`,`macos` have more or less the same content as now, only split in various pages
  - `edge` describes installing from github
  - `other` has info on docker, are there other platforms to add?

I tried to follow the style that was present, I'm not that experienced though so feedback welcome.
Ps: I did not update download links, shas etc already present, as I concentrated on layout, I think in the future we should use a variable in _config.yml so we can update them all once.

### CODE REVIEW

File: jekyll_prototype/_config.yml [hunk 1]
```diff
@@ -14,3 +14,5 @@ exclude:
   - CNAME
   - README.md
   - LICENSE.md
+host: 0.0.0.0
```
Comment (dom96) [inline jekyll_prototype/_config.yml]: Please remove this. We don't want to serve it globally by default and I'm already used to the default 4000 port.

Comment (stisa) [inline jekyll_prototype/_config.yml]: Done, I used it to test on my local machines and forgot to remove it.

**Comment by dom96** [comment]
Is ``jekyll_prototype/assets/bountysource/bountysource.png`` needed?

**Comment by stisa** [comment]
Nope, the only needed ones are the two logos, `xored.svg` and `secondspectrum.png`. 
I thought they could be reused in the future, do you want me to remove them for now, leaving only logos?

File: jekyll_prototype/_layouts/download.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  {% include head.html %}
+  <body class="site">
+    {% include header.html %}
+    <div class="site-content">
+      {% include downloadtop.html %}
```
Comment (dom96) [inline jekyll_prototype/_layouts/download.html]: Is this downloadtop.html and bottom really necessary?

**Comment by dom96** [comment]
Yes, please remove them.

**Comment by dom96** [comment]
All in all, thank you for taking the time to do this. I do think that the downloads pages will change significantly (I won't to make it as simple as possible to get started with Nim), but the sponsors page looks perfect.

**Comment by stisa** [comment]
Done and done, yeah the download pages don't really impress me either, they could use some refinement.

**Comment by dom96** [comment]
Thanks :)

**Pull request merged**